ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt

```diff
@@ -102,4 +107,22 @@ class MainService : Service() {
     override fun onDestroy() {
         Toast.makeText(this, "The MainService has been gone", Toast.LENGTH_SHORT).show()
     }
+
+    private fun initNNStreamer() {
+        if (this.initialized) {
+            return
+        }
+        try {
+            initialized = NNStreamer.initialize(this)
+        } catch (e: Exception) {
+            e.printStackTrace()
+            Log.e(TAG, e.message!!)
+        } finally {
+            if (initialized) {
+                Log.i(TAG, "Version: " + NNStreamer.getVersion())
+            } else {
+                Log.e(TAG, "Failed to initialize NNStreamer")
+            }
+        }
+    }
```

Comment:
Everything looks good to me..

Anyway... the "stop" mark at the end of this file is... a bit alarming..